docs/sources/source/nlp/bert_pretraining.rst

```diff
@@ -261,20 +262,6 @@ Next, we define necessary callbacks:
 2. `EvaluatorCallback`: tracking metrics during evaluation at set intervals
 3. `CheckpointCallback`: saving model checkpoints at set intervals
 
-    .. code-block:: python
-
-        train_callback = nemo.core.SimpleLossLoggerCallback(tensors=[train_loss],
-            print_func=lambda x: print("Loss: {:.3f}".format(x[0].item())))),
-            step_freq=args.train_step_freq,
-        eval_callback = nemo.core.EvaluatorCallback(eval_tensors=[eval_loss],
-            user_iter_callback=nemo_nlp.callbacks.lm_bert_callback.eval_iter_callback,
-            user_epochs_done_callback=nemo_nlp.callbacks.lm_bert_callback.eval_epochs_done_callback
-            eval_step=args.eval_step_freq)
-        ckpt_callback = nemo.core.CheckpointCallback(folder=nf.checkpoint_dir,
-            epoch_freq=args.save_epoch_freq,
-            load_from_folder=args.load_dir,
-            step_freq=args.save_step_freq)
-
```

Comment:
why are you removing callbacks?

Reply:
Thought this is pretty standard for every model, so did not want to overload the tutorial for this model specifically. If people want to know details, they can refer to the example or notebook. What do you think?